modules/installation-adding-registry-pull-secret.adoc

```diff
@@ -99,18 +99,54 @@ The contents of the file resemble the following example:
 ----
 // An additional step for following this procedure when using oc-mirror as part of the disconnected install process.
 ifdef::oc-mirror[]
-. Save the file either as `~/.docker/config.json` or `$XDG_RUNTIME_DIR/containers/auth.json`.
+. Save the file as either `~/.docker/config.json` or `$XDG_RUNTIME_DIR/containers/auth.json`. 
+
+.. If the `.docker` or `$XDG_RUNTIME_DIR/containers` directories do not exist, create one by entering the following command:
++
+[source,terminal]
+----
+$ mkdir -p <directory_name>
+----
++
+Where `<directory_name>` is either `~/.docker` or `$XDG_RUNTIME_DIR/containers`.
+
+.. Copy the pull secret to the appropriate directory by entering the following command:
++
+[source,terminal]
+----
+$ cp <path>/<pull_secret_file_in_json> or <directory_name>/<auth_file>
+----
++
+Where `<directory_name>` is either `~/.docker` or `$XDG_RUNTIME_DIR/containers`, and `<auth_file>` is either `config.json` or` auth.json`.
 endif::[]
 
 ifdef::oc-mirror-v2[]
 . Save the file as `$XDG_RUNTIME_DIR/containers/auth.json`.
 endif::[]
 // Similar to the additional step above, except it is framed as optional because it is included in a disconnected update page (where users may or may not use oc-mirror for their process)
 ifdef::update-oc-mirror[]
-. Optional: If using the oc-mirror plugin, save the file either as `~/.docker/config.json` or `$XDG_RUNTIME_DIR/containers/auth.json`.
+. Optional: If using the oc-mirror plugin, save the file as either `~/.docker/config.json` or `$XDG_RUNTIME_DIR/containers/auth.json`. 
+
+.. If the `.docker` or `$XDG_RUNTIME_DIR/containers` directories do not exist, create one by entering the following command:
++
+[source,terminal]
+----
+$ mkdir -p <directory_name>
+----
++
+Where `<directory_name>` is either `~/.docker` or `$XDG_RUNTIME_DIR/containers`.
+
+.. Copy the pull secret to the appropriate directory by entering the following command:
++
+[source,terminal]
+----
+$ cp <path>/<pull_secret_file_in_json> or <directory_name>/<auth_file>
+----
++
+Where `<directory_name>` is either `~/.docker` or `$XDG_RUNTIME_DIR/containers`, and `<auth_file>` is either `config.json` or` auth.json`.
 endif::[]
```

Comment:
are there two still open conditionals above these? wonder if the new one made them out of sequence?

Reply:
yeah one of these matches ifndef::openshift-origin[], which is further up than this PR goes. 

I've removed that content and can't get the issue to resolves. Ay yai yai.